examples/vweb_orm_jwt/.env.default

```diff
@@ -0,0 +1,6 @@
+DB_USERNAME=
+DB_PASSWORD=
+DB_HOST=
+DB_PORT=
+DB_NAME=
+SECRET_KEY=
```

Comment:
hm, probably remove this file as well; it is not used by V or the builtin modules, and users will definitely have other values locally, so it may be confusing/creating conditions for merge conflicts, if they decide to change these lines.